### DISCUSSION TITLE
Implants?

### DISCUSSION THREAD

**Discussion by paintballlaws** [Wilm0rien/EveOnlineMiningYield#4] [Ideas]
Any thoughts on adding the values with either 1 or both of the mining implants?

**Comment by Wilm0rien** [comment]
yes! all values are already including the 5% 'Highwall' / 'Yeti' implants! you can see this in the calculation formula (e.g. $Highwall_Mining_MX1005_Implant). I did not add the 'Michi' implant because it just adds another 5% overall bonus and it is too expensive (you can buy a mining fleet for this, including an orca) to be used by many players. 

it is a valid question ask: is it worthwhile to buy a 2b implant for mining and how long do I have to mine to charge it off. 

when you spend 6b to plug three 'michi' implants in the three hulk toons in the mining fleet example (3 hulks + rorqual) 
- the outcome would be that the fleet will finish the cluster in 84min instead of 88min. 
- if mining bistot you will get an additional 14m isk for this extra 4min 
- so you have to do this 428 times to get enough extra isk for the 6b implants to charge off, 
- this will take 628 hours to complete. 

I don't think its is a realistic scenario that some one finishes the 628h without loosing at least one hulk while mining in null sec.
